actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -437,6 +442,9 @@ def link_to_if(condition, name, options = {}, html_options = {}, &block)
       # Creates a mailto link tag to the specified +email_address+, which is
       # also used as the name of the link unless +name+ is specified. Additional
       # HTML attributes for the link can be passed in +html_options+.
+      # Note the order of params, which is reversed compared to +link_to+. Provide
+      # the part that is used for the href first, and the part that is visible
+      # to users second.
```

Comment:
Do you think that is necessary to point out? It seems pretty obvious from the docs. Also the case where the link target (email address) is also the visible text is way more likely in the case of `mail_to`. That's probably why it's reversed in the first place. To make the text more optional.